etc/ReplayOfflineConfiguration.py

```diff
@@ -1519,8 +1519,7 @@
 ###################################
 ### configure memory for stream ###
 ###################################
-setStreamMem(tier0Config,"NanoDST",value=2000)
-setStreamMem(tier0Config,"ScoutingPF",value=2000)
+setStreamMem(tier0Config, NanoDST = 2000, ScoutingPF = 2000)
```

Comment:
As I noted in Tier0config.py, you are not using *streamid. Maybe you could explain the use case that this list covers.